airlock/templates/file_browser/file_content/csv.html

```diff
@@ -207,7 +207,7 @@
             </table>
           </div>
         </div>
-        {% vite_asset "assets/src/scripts/csv-datatable.js" %}
+        {% vite_asset "assets/src/scripts/clusterize-datatable.js" %}
```

Comment:
This is fine, although it was actually the "datatable" bit that I found confunsingly named, rather than the "csv" bit. But it's clear now that it's a clusterize datatable we're talking about rather than a datatables datatable, and if we're planning at somepoint to change the directory table to use this, it makes sense that it isn't labelled with csv.

Reply:
Ha - ok. Yes I was making it (1) not csv-specific, and (2) explicitly clusterize. But you're right about the datatable bit, probably clusterize-table would have been better. I may implement it for the file-browser if I have time this week and can change it then.